.github/workflows/test-multiple-builds.yml

```
@@ -12,24 +12,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        build: [cjs, umd] # [cjs, esm, umd, system]
-        env: [development, production]
-        devtools-skip:
-          - CI-MATRIX-NOSKIP
-        include:
-          - devtools-skip: CI-MATRIX-[2345]
-            build: umd
-          - devtools-skip: CI-MATRIX-[1345]
-            build: umd
-          - devtools-skip: CI-MATRIX-[1245]
-            build: umd
-          - devtools-skip: CI-MATRIX-[1235]
-            build: umd
-          - devtools-skip: CI-MATRIX-[1234]
-            build: umd
-        exclude:
-          - devtools-skip: CI-MATRIX-NOSKIP
-            build: umd
+        build: [cjs, esm, umd] # [cjs, esm, umd, system]
+        env: [development] # [development, production]
```

@dai-shi, I copied this change from https://github.com/pmndrs/valtio/pull/819/files, but looking at this, could it be a mistake? I'd like to test both `development` and `production` in the testing matrix. Should this be changed to the following?
```suggestion
        env: [development, production]
```